bin/omarchy-toggle-nightlight

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Default temperature values
+ON_TEMP=4000
+OFF_TEMP=6500
+
+# Query the current temperature
+CURRENT_TEMP=$(hyprctl hyprsunset temperature 2>/dev/null | grep -oE '[0-9]+')
+
+if [[ "$CURRENT_TEMP" == "$OFF_TEMP" ]]; then
+  hyprctl hyprsunset temperature $ON_TEMP
+  omarchy-restart-waybar # restart waybar in case user has waybar module for hyprsunset
+  notify-send "  Nightlight On"
+else
+  hyprctl hyprsunset temperature $OFF_TEMP
+  omarchy-restart-waybar # restart waybar in case user has waybar module for hyprsunset
```

Comment:
I'm on the fence about this. Because the problem with restarting waybar is that it gives an ugly jolt. Really don't love that as part of the default setup.

One option could be to grep the waybar config to look for hyprsunset, though. Only restart if that's found.